modules/friendlytag.js

```diff
@@ -203,9 +166,19 @@ Twinkle.tag.callback = function friendlytagCallback() {
 	Window.setContent(result);
 	Window.display();
 
-	if (Twinkle.tag.mode === 'article') {
+	// for quick filter:
+	$allCheckboxDivs = $(result).find('[name$=Tags]').parent();
+	$allHeaders = $(result).find('h5');
+	result.quickfilter.focus();  // place cursor in the quick filter field as soon as window is opened
+	result.quickfilter.autocomplete = 'off'; // disable browser suggestions
+	result.quickfilter.addEventListener('keypress', function(e) {
+		if (e.keyCode === 13) { // prevent enter key from accidentally submitting the form
+			e.preventDefault();
+			return false;
+		}
+	});
```

Comment:
Ahh, should've looked at this before merging #668, but it's an easy merge conflict so no harm.  Will try to review this later today/tomorrow morning.